xwalk_core_library_android.gypi

```diff
@@ -466,8 +472,11 @@
           ],
           'action': [
             'python', '<(DEPTH)/xwalk/build/android/generate_xwalk_core_library_aar.py',
-            '-t', '<(PRODUCT_DIR)',
-            '--shared',
+            '--aar-path', '<(PRODUCT_DIR)/xwalk_shared_library.aar',
+            '--android-manifest', '<(DEPTH)/xwalk/build/android/xwalk_shared_library_template/AndroidManifest.xml',
+            '--classes-jar', '<(PRODUCT_DIR)/lib.java/xwalk_core_library_java_app_part.jar',
+            '--res-dir', '<(PRODUCT_DIR)/xwalk_shared_library/res',
+            '--r-txt', '<(PRODUCT_DIR)/xwalk_core_internal_empty_embedder_apk/gen/R.txt',
```

Comment:
The "always_run" version is a hack that the previous people who worked on this file came up with -- you can see it's not present in Chromium itself. It basically points to a file that is never created, so the target is always run regardless of whether it needs to.
The stamp version is much more commonly used -- you pass a file that will be touched when the script finishes running, and the build system uses that to determine whether the inputs are more recent that it and only call the script if something changed.